From a4f63404cd0e6deb8e771bd5ebf9bc52b398e8d5 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 17 Jun 2023 16:39:56 +0200
Subject: [PATCH] Dpkg::Source::Package::V2: Mark single-debian-patch as not
 needing forwarding

These patches are autogenerated and in general should not be submitted
upstream as is, as they are going to be a conglomerate of unrelated
changes that need to be submitted upstream in atomic pieces. So applying
the patch tagging guidelines in this situation is not very helpful.

Closes: #1038121
---
 scripts/Dpkg/Source/Package/V2.pm | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/scripts/Dpkg/Source/Package/V2.pm b/scripts/Dpkg/Source/Package/V2.pm
index 4aad9ba8c..1f0946128 100644
--- a/scripts/Dpkg/Source/Package/V2.pm
+++ b/scripts/Dpkg/Source/Package/V2.pm
@@ -621,11 +621,13 @@ sub _get_patch_header {
 
     if ($self->{options}->{single_debian_patch}) {
         return <<'AUTOGEN_HEADER';
-This is an autogenerated patch header for a single-debian-patch file. The
-delta against upstream is either kept as a single patch, or maintained
-in some VCS, and exported as a single patch instead of more manageable
-atomic patches.
+Description: Autogenerated patch header for a single-debian-patch file.
+ The delta against upstream is either kept as a single patch, or maintained
+ in some VCS, and exported as a single patch instead of more manageable
+ atomic patches.
+Forwarded: not-needed
 
+---
 AUTOGEN_HEADER
     }
 
-- 
2.40.1

